dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormField.vue

```diff
@@ -0,0 +1,163 @@
+                v-model="currentPageValue"
+                type="number"
+                name="page"
+                placeholder="Page"
+                validation-label="Page"
+                step="1"
+                min="0"
+                validation="min:0"
+              />
+            </div>
+          </div>
+        </FormKit>
+      </div>
+
+      <!-- Data quality -->
+      <div class="mb-4" data-test="dataQuality">
+        <UploadFormHeader
+          :label="`${label} Quality`"
+          description="The level of confidence associated to the value."
+          :is-required="isDataQualityRequired"
+        />
+        <div class="md:col-6 col-12 p-0">
+          <FormKit
+            type="select"
+            :modelValue="!isDataQualityRequired ? 'NA' : currentQualityValue"
+            name="quality"
+            :validation="isDataQualityRequired ? 'required' : ''"
+            validation-label="Data quality"
+            placeholder="Data quality"
+            :options="qualityOptions"
+          />
+        </div>
+      </div>
+      <div class="form-field">
+        <FormKit
+          type="textarea"
+          name="comment"
+          placeholder="(Optional) Add comment that might help Quality Assurance to approve the datapoint. "
+        />
+      </div>
+    </FormKit>
+  </template>
+
+  <template v-else>
+    <UploadFormHeader :label="label" :description="description ?? ''" :is-required="required" />
+    <FormKit
+      type="text"
+      :name="name"
+      v-model="currentValue"
+      :validation-label="validationLabel ?? label"
+      :validation="`number|${validation}`"
+      :placeholder="placeholder"
+      :inner-class="innerClass"
+    />
+  </template>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import UploadFormHeader from "@/components/forms/parts/elements/basic/UploadFormHeader.vue";
+import { FormKit } from "@formkit/vue";
+import { QualityOptions } from "@clients/backend";
+import { selectNothingIfNotExistsFormKitPlugin } from "@/utils/FormKitPlugins";
+import { YesNoFormFieldProps } from "@/components/forms/parts/fields/FormFieldProps";
+
+export default defineComponent({
+  name: "DataPointFormField",
+  components: { UploadFormHeader, FormKit },
+  inject: {
+    injectReportsName: {
+      from: "namesOfAllCompanyReportsForTheDataset",
+      default: [] as string[],
+    },
+  },
+  computed: {
+    referenceableReportNames(): string[] {
+      return (this.reportsName ?? this.injectReportsName) as string[];
+    },
+    isDataQualityRequired(): boolean {
+      return this.currentValue !== "";
+    },
+  },
+  data() {
+    return {
+      qualityOptions: Object.values(QualityOptions).map((qualityOption: string) => ({
+        label: qualityOption,
+        value: qualityOption,
+      })),
+      currentValue: "",
+      currentUnit: "",
+      currentReportValue: "",
+      currentPageValue: "",
+      currentQualityValue: "",
```

Comment:
It seems that you can remove these (also from the DOM) and it still works, since you never really run code on those values:

![image](https://github.com/d-fine/Dataland/assets/99255296/3847afac-c684-4b41-9773-a222bae5a360)

Reply:
Removed and waiting to see if the CI remains green

Reply:
Tests are green

Reply:
Removed the three occurrences

Reply:
👍